sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -168,14 +168,30 @@ private void cancelTimer() {
     return children;
   }
 
+  /**
+   * @deprecated use {@link getStartDate()} instead as it offers better precision
+   */
+  @Deprecated
```

Comment:
L: This class is already annotated with internal, so you can remove the methods directly